### PR TITLE
Fix/cta text link

### PR DESCRIPTION
**Pull request by kennylam**
### Related Ticket(s)

#1682 

### Description

Fixes `href` prop for `CTA` component when using `text` style

### Changelog

**New**

- add `style` and `type` CTA knobs to `ContentBlockSimple` story

**Changed**

- fix `href` prop for `text` style CTA

### CODE REVIEW

**Comment by jeffchew** [comment]
@kennylam the following stories look busted:

- ContentBlockMixed
- Content Item
- pictogram item
- Card
- CTA

File: packages/react/src/patterns/blocks/ContentBlockMixed/__stories__/ContentBlockMixed.stories.js [hunk 1]
```diff
@@ -17,13 +17,24 @@ storiesOf('Patterns (Blocks)|ContentBlockMixed', module)
   })
   .add('Default', () => {
     const heading = 'Lorem ipsum dolor sit amet';
+    const ctaStyles = {
+      text: 'text',
+      card: 'card',
```
Comment (annawen1) [inline packages/react/src/patterns/blocks/ContentBlockMixed/__stories__/ContentBlockMixed.stories.js]: I think we only allow card cta style for the contentblockmixed pattern

https://ibm.ent.box.com/file/597369069325

**Comment by annawen1** [comment]
looks good besides reverting the content block mixed knobs

File: packages/react/src/patterns/blocks/ContentBlockMixed/__stories__/ContentBlockMixed.stories.js [hunk 2]
```diff
@@ -17,13 +17,24 @@ storiesOf('Patterns (Blocks)|ContentBlockMixed', module)
...
 
-    const cta = {
+    const ctaProps = {
       cta: {
-        href: 'https://www.example.com',
+        href: 'https://www.ibm.com',
       },
-      style: 'card',
-      type: 'local',
+      style: select('CTA style', ctaStyles, ctaStyles.card),
```
Comment (jeffchew) [inline packages/react/src/patterns/blocks/ContentBlockMixed/__stories__/ContentBlockMixed.stories.js]:
```suggestion
      style: 'card',
```

File: packages/react/src/patterns/blocks/ContentBlockMixed/__stories__/ContentBlockMixed.stories.js [hunk 3]
```diff
@@ -17,13 +17,24 @@ storiesOf('Patterns (Blocks)|ContentBlockMixed', module)
   })
   .add('Default', () => {
     const heading = 'Lorem ipsum dolor sit amet';
+    const ctaStyles = {
+      text: 'text',
+      card: 'card',
+    };
```
Comment (jeffchew) [inline packages/react/src/patterns/blocks/ContentBlockMixed/__stories__/ContentBlockMixed.stories.js]:
```suggestion
```

File: packages/react/src/patterns/blocks/ContentBlockMixed/__stories__/ContentBlockMixed.stories.js (hunk 1 above)
Comment (jeffchew) [reply packages/react/src/patterns/blocks/ContentBlockMixed/__stories__/ContentBlockMixed.stories.js]: Was able to adjust via PR suggestions

**Pull request merged**